_posts/2024-04-03-UsingDocumentationForUsabilityStudies.md

```diff
@@ -0,0 +1,107 @@
+In your instructions make sure to remind the participant to talk out loud and tell you what they are thinking. This is very effective in reviewing usage behavior as it is very difficult to discern what a user is thinking if they do not talk out loud. After the study participant has attempted to accomplish the task, add follow up questions on the experience that they just went through. These questions can range from whether the participant thought they were successful, to suggestions for improvements in the task that they just accomplished. Given the time it takes to do a usability study, it is recommended that each study entails 4-6 complex tasks or 8-10 simple tasks. More tasks will make the study too long to complete. Once you are satisfied with the task that you will want your study participant to complete, think through any final questions that you will want to ask the participant regarding the entire experience or the software as a whole. See this template and example of a usability study plan including a recruitment email.
+![](/assets/media/blog-images/2024-04-03-UsingDocumentationForUsabilityStudies/Screenshot 2024-03-26 at 10.42.52 AM.png)
+
+### What do you need to start a usability study?
+
+There are some convenient tooling such as UserZoom (now UserTesting) or Lookback and others. These tools make it convenient to capture user screens, record their voice and facial expressions and integrate surveys after tasks. They also offer a variety of testing methodologies to incorporate in the design testing phase of product development. That said, a usability study can be conducted manually over any video conferencing software (such as Zoom). If no specialized tooling is used, make sure to record all calls, ask the user to screen share and to ask probing questions as relevant. If you automate your study to be run with just the prompts the software provides and you do not need to be present, it is called an unmoderated study. If on the other hand, you spend the time to walk through the experience and help with the questions and probing, it is called a moderated study. Moderated studies are more time consuming at the data collection stage, but can be an effective method for rapid prototyping if the team plans to incrementally and iteratively improve on the design. 
+
+
+### Sample considerations
+
+When you run your study, sample considerations are important. Use you persona definitions to guide sample specifications for recruitment. For larger samples and basic usability concerns, consider obtaining a paid sample. Use your screener to qualify or disqualify participants for your study. If your experience is for a niche set of users, it will make sense to recruit them by leveraging your B2B sales and services partners. For open source projects consider blogging, dropping links on GitHub, your community forum, announcing on community meetings etc. Always validate your sample with the screening questions either to 
```

Comment:
```suggestion
When you run your study, sample considerations are important. Use your persona definitions to guide sample specifications for recruitment. For larger samples and basic usability concerns, consider obtaining a paid sample. Use your screener to qualify or disqualify participants for your study. If your experience is for a niche set of users, it will make sense to recruit them by leveraging your B2B sales and services partners.
```